plugin/src/main/java/com/denizenscript/denizen/objects/properties/entity/EntityEquipmentDropChance.java

```diff
@@ -0,0 +1,96 @@
+package com.denizenscript.denizen.objects.properties.entity;
+
+import com.denizenscript.denizen.objects.EntityTag;
+import com.denizenscript.denizencore.objects.Mechanism;
+import com.denizenscript.denizencore.objects.core.ElementTag;
+import com.denizenscript.denizencore.objects.core.MapTag;
+import org.bukkit.inventory.EntityEquipment;
+
+public class EntityEquipmentDropChance extends EntityProperty<MapTag> {
+
+    // <--[property]
+    // @object EntityTag
+    // @name equipment_drop_chance
+    // @input MapTag
+    // @description
+    // Controls the chance of each piece of equipment dropping when the entity dies.
+    // Specifying a drop chance of 0 will prevent the item from dropping, a drop chance of 1 will always drop the item if killed by a player, and a drop chance of higher than 1 will always drop the item no matter what the entity was killed by.
+    //
+    // @Usage
+    // Use to prevent a zombie from dropping any of its equipped items, no matter what:
+    // - adjust <[zombie]> equipment_drop_chance:[helmet=0;chestplate=0;leggings=0;boots=0;main_hand=0;off_hand=0]
+    //
+    // -->
+
+    public static boolean describes(EntityTag entity) {
+        return true;

```

Comment:
I believe this is living entity only?

Reply:
Actually looking at the Javadocs this seems to be limited to `Mob`s
<img width="576" alt="image" src="https://github.com/DenizenScript/Denizen/assets/31237389/694b5e03-aaa9-48c3-9557-712e51bd29cd">